tools/scripts/xabuild

```diff
@@ -52,11 +52,24 @@ if [ -z "$CONFIGURATION" ]; then
 		esac
 
 		if [ -n "$CONFIGURATION" ]; then
+			for word in $CONFIGURATION; do
+				CONFIGURATION=$word
+				break
+			done
+
```

Comment:
I fixed a slight issue here. If you run the command:
```
tools/scripts/xabuild /p:Configuration=Release someProjectOrSolution.csproj
```

Then `$CONFIGURATION` was coming out set to `Release someProjectOrSolution.csproj`.

Let me know if there is a better way to fix this, I'm not exactly a bash expert. 😄

Reply:
I think the actual problem is line 46: it should use `"$@"`, not `"$*"`:

https://unix.stackexchange.com/a/129077